Code/Framework/AzCore/AzCore/Settings/SettingsRegistryConsoleUtils.cpp

```diff
@@ -122,6 +123,38 @@ namespace AZ::SettingsRegistryConsoleUtils
         }
     }
 
+    static void ConsoleDumpSettingsFileOriginValue(SettingsRegistryOriginTracker& settingsRegistryOriginTracker, const ConsoleCommandContainer& commandArgs)
+    {
+        AZStd::string outputString;
+        for (AZStd::string_view settingsKeyToDump : (commandArgs.empty() ? AZStd::vector<AZStd::string_view>{""} : commandArgs))
+        {
+            settingsRegistryOriginTracker.VisitOrigins(settingsKeyToDump, [&outputString](const AZ::SettingsRegistryOriginTracker::SettingsRegistryOrigin& settingsRegistryOrigin)
+            {
+                outputString += AZStd::string::format("Key: %s Origin: %s Value when Merged: %s\n", settingsRegistryOrigin.m_settingsKey.c_str(),
```

Comment:
You may want to put the Key and path in double quotes, so that we know when the root key is being shown.
```suggestion
                outputString += AZStd::string::format("Key: \"%s\" Origin: \"%s\" Value when Merged: %s\n", settingsRegistryOrigin.m_settingsKey.c_str(),
```